src/bin/cli/mod.rs

```diff
@@ -682,3 +697,211 @@ fn explicit_args(matches: &ArgMatches) -> Vec<&Id> {
         })
         .collect()
 }
+
+/// A helper function to check if the target name provided by the user matches the target name in solang.toml
+/// Returns false if the target names do not match
+/// Returns true if the target names match or if solang.toml does not exist in the current directory
```

Comment:
This is kind of obvious, what's not so obvious is under what conditions it will fail and return `Err()` instead. Can you please change the comment to explain this?